crates/circuit/src/dag_circuit.rs

```diff
@@ -758,7 +759,7 @@ def _format(operand):
     /// Raises:
     ///     DAGCircuitError: a clbit is not a :obj:`.Clbit`, is not in the circuit,
     ///         or is not idle.
-    #[pyo3(signature = (*clbits))]
+    #[pyo3(signature = (* clbits))]
```

Comment:
not sure why these signatures got formatted... but I can revert them once the PR is ready to merge.